File: php-sdk/src/triplink/vcc/card/client/query/SettlementInfoResp.php
<?php
/**
 * Created by PhpStorm.
 * User: wang.xiaoliang
 * Date: 2021/8/5
 * Time: 13:57
 */
namespace triplink\vcc\card\client\query;


class SettlementInfoResp
{
    /**
     * 返回总条数
     */
    var $totalCount;

    /**
     * 可用余额
     */
    var $balanceAmt;

    /**
     * 清算交易记录
     */
    var $settlementInfos;

    /**
     * @return mixed
     */
    public function getTotalCount()
    {
        return $this->totalCount;
    }

    /**
     * @param mixed $totalCount
     */
    public function setTotalCount($totalCount): void
    {
        $this->totalCount = $totalCount;
    }

    /**
     * @return mixed
     */
    public function getBalanceAmt()
    {
        return $this->balanceAmt;
    }

    /**
     * @param mixed $balanceAmt
     */
    public function setBalanceAmt($balanceAmt): void
    {
        $this->balanceAmt = $balanceAmt;
    }

    /**
     * @return mixed
     */
    public function getSettlementInfos()
    {
        return $this->settlementInfos;
    }

    /**
     * @param mixed $settlementInfos
     */
    public function setSettlementInfos($settlementInfos): void
    {
        $this->settlementInfos = $settlementInfos;
    }


}

?>

File: src/main/java/com/ctrip/ccard/creditcard/vcc/bean/V1/SettlementInfoResp.java
package com.ctrip.ccard.creditcard.vcc.bean.V1;

import java.util.List;

/**
 * Description:
 */
public class SettlementInfoResp {
    /**
     * 返回总条数
     */
    private Integer totalCount;

    /**
     * 可用余额
     */
    private String balanceAmt;

    /**
     * 清算交易记录
     */
    private List<SettlementInfo> settlementInfos;

    public Integer getTotalCount() {
        return totalCount;
    }

    public void setTotalCount(Integer totalCount) {
        this.totalCount = totalCount;
    }

    public String getBalanceAmt() {
        return balanceAmt;
    }

    public void setBalanceAmt(String balanceAmt) {
        this.balanceAmt = balanceAmt;
    }

    public List<SettlementInfo> getSettlementInfos() {
        return settlementInfos;
    }

    public void setSettlementInfos(List<SettlementInfo> settlementInfos) {
        this.settlementInfos = settlementInfos;
    }
}
